app.production.js

```diff
@@ -1,16 +1,14 @@
 const htmlStandards = require('reshape-standard')
 const cssStandards = require('spike-css-standards')
-const {UglifyJsPlugin} = require('webpack').optimize
+const optimize = require('spike-optimize')
```

Comment:
Spike Optimize is a fresh new plugin that has become the default for new spike projects. It is potentially the simplest interface for an advanced webpack optimization tool that exists. It will activate scope hoising, minification, aggressive splitting, chunkhash renaming, and the common chunk plugin, then go through your view files and rename/re-format any script tags automatically to match the newly renamed and potentially split scripts :grin: